coopr-server/src/main/java/co/cask/coopr/codec/json/current/LeaseDurationCodec.java

```diff
@@ -34,11 +34,12 @@ public LeaseDuration deserialize(JsonElement json, Type typeOfT, JsonDeserializa
     throws JsonParseException {
     JsonObject jsonObj = json.getAsJsonObject();
 
-    Long initial = context.deserialize(jsonObj.get("initial"), Long.class);
-    Long max = context.deserialize(jsonObj.get("max"), Long.class);
-    Long step = context.deserialize(jsonObj.get("step"), Long.class);
+    LeaseDuration.Builder builder = LeaseDuration.builder();
+    builder.setInitial(jsonObj.getAsJsonPrimitive("initial").getAsString());
```

Comment:
I think, but am not sure, that this fails with a NPE if there is no "initial" key, which is why ``` context.deserialize(jsonObj.get("initial"), String.class``` is used in other places. We should add a test case where one is missing to see how it behaves and make sure we handle it correctly.